src/curated-corpus/components/ScheduleItemForm/ScheduleItemForm.tsx

```diff
@@ -208,8 +212,34 @@ export const ScheduleItemForm: React.FC<
         section.status !== SectionStatus.Expired,
     ) || [];
 
+  const getAdapterLocale = (locale: string): string =>
+    locale.split('-')[0]?.toLowerCase() ?? 'en';
+
+  const activeSurfaceGuid =
+    formik.values.scheduledSurfaceGuid ||
+    scheduledSurfaceGuid ||
+    (scheduledSurfaces.length === 1 ? scheduledSurfaces[0].guid : undefined);
```

Comment:
i think you can simplify this to use `selectedScheduledSurfaceGuid` (which is defined above):

```
const activeSurfaceGuid =
    formik.values.scheduledSurfaceGuid || selectedScheduledSurfaceGuid;
```